src/client/client.cpp

```diff
@@ -1244,8 +1244,12 @@ void Client::sendPlayerPos()
 	u8 camera_fov    = map.getCameraFov();
 	u8 wanted_range  = map.getControl().wanted_range;
 
-	// Save bandwidth by only updating position when something changed
-	if(myplayer->last_position        == myplayer->getPosition() &&
+	// Save bandwidth by only updating position when
+	// player is not dead and something changed
+	if (myplayer->isDead())
+		return;
```

Comment:
Not sure about this, if the player dies while falling or while travelling in a vehicle, the body should continue to move. I suspect this change might cause new bugs. It's also not necessary.
If death prevents autoforward happening then the player will often be stationary when dead and line 1252 takes care of this.
I think player movement while dead should be considered separately from this PR, there are some issues about this already.

Reply:
@paramat The server already ignores this packet when the player is dead. This PR just prevents the client from sending the packet that is bound to be ignored by the server anyway. See

https://github.com/minetest/minetest/blob/72feab081c336d2d71d300131d30e71694b485f3/src/network/serverpackethandler.cpp#L518-L523

Reply:
Do dead players still fall down (from the dead player's perspective / from other players' perspective; currently in master / with this PR)?

Reply:
They do see themselves fall. I can't say for sure if players can see other dead players' corpses moving after they die, as I haven't been in that situation yet. Once they die, their position remains the same. This also explains why bones sometimes spawn midair if player dies of fall damage - the player has already fallen and died, but the server knows the player died before the player's PLAYER_POS packet reaches it. The player's position is taken to be the one from the last sent packet until they respawn.

Reply:
Ok thanks for explaining, i have no issue with then.